src/include/sof/platform.h

```diff
@@ -54,6 +54,14 @@ int platform_context_save(struct sof *sof);
 
 /** @}*/
 
+#ifdef CONFIG_IMX8ULP
```

Comment:
I would also put this in platform/memory.h since mappings are likely to change over time and platform.